src/Components/Carousel/index.js

```diff
@@ -0,0 +1,71 @@
+import React, { Component } from 'react';
+import CarouselFrame from './CarouselFrame';
+import './index.scss';
+
+export default class Carousel extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      listTransform: 0,
+      listTransition: '1s ease-in-out',
+    };
+  }
+
+  moveToNext = () => {
+    if (this.state.listTransform > -640 * 4) {
+      this.setState({
+        listTransform: this.state.listTransform - 640,
+        listTransition: '1s ease-in-out',
+      });
+    } else {
+      this.setState({
+        listTransform: 0,
+        listTransition: '0s',
+      });
+    }
+  };
+
+  moveToPre = () => {
```

Comment:
```suggestion
  moveToPrev = () => {
```
보통 이전 값을 지칭할 때 prev라고 많이 쓰기 때문에 여기도 그렇게 써주면 좋을거 같아요